python/cudf/cudf/core/index.py

```diff
@@ -97,7 +97,20 @@ def deserialize(cls, header, frames):
 
     @property
     def names(self):
-        return (self.name,)
+        return [self.name]
```

Comment:
Should this still be a `tuple` in the rare event someone tries to assign to this later (without the setter being involved)? Would help catch any errors in that event 🙂

Reply:
This actually can't be a tuple as trying to set the names of an index via a tuple is not allowed to protect against the case where the name is actually a tuple.

Reply:
Pandas uses their own `FrozenList` for this reason but it felt weird to reach deep inside of Pandas to use that here.

Reply:
I was actually incorrect here and misread an exception from passing a string directly as opposed to a tuple 😄